config.go

```diff
@@ -116,6 +117,10 @@ func initConfig() error {
 			return err
 		}
 	}
+
+	// Update config from environment variables.
+	processEnv()
```

Comment:
Are you suggesting that the config file should override the environment variables?